packages/modelence/src/bin/deploy.ts

```diff
@@ -97,12 +97,17 @@ async function createBundle(bundlePath: string) {
   console.log(`Deployment bundle created at: ${bundlePath} (${(stats.size / 1024 / 1024).toFixed(2)} MB)`);
 }
 
-async function uploadBundle(environmentAlias: string, bundlePath: string, token: string) {
-  const response = await fetch(getStudioUrl(`/api/environments/${environmentAlias}/upload`), {
+async function uploadBundle(appAlias: string, envAlias: string, bundlePath: string, token: string) {
+  const response = await fetch(getStudioUrl(`/api/upload`), {
```

Comment:
Can we rename this to `upload-bundle`? Now that we've removed the rest of the path `/api/upload` is too generic and could refer to any upload (in the future we can have other assets to upload).